src/home/components/SignupFrame/SignupFrame.style.ts

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+export const StyledSignupFrame = styled.div`
+  width: fit-content;
+  flex: 1; // vertical centering
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 22px;
+`;
+
+export const StyledSignupFrameLogo = styled.img`
+  width: 180px;
+`;
+
+export const StyledSignupFrameContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+
+  width: 480px;
+  padding: 22px 24px 22px 24px;
```

Comment:
패딩 값이 상하는 22px, 좌우는 24px로 들어가니까 줄여쓸 수 있을 거 같아요~! ([참고 문서](https://developer.mozilla.org/ko/docs/Web/CSS/padding))

```suggestion
  padding: 22px 24px;
```

Reply:
감사합니다!
[여기](https://github.com/yourssu/Soomsil-Web/pull/138/commits/ffeb4ea0a5ef3a4b70792b59d9932fd67d677181)에서 리팩토링 진행했습니다!